ansible/roles/openshift_udp_limits/files/iptables_udp.conf

```diff
@@ -1,5 +1,3 @@
 # Ansible-managed
-:msg, startswith, "UDP:" -/var/log/iptables.log
-& stop
-:msg, regex, "^\[ *[0-9]*\.[0-9]*\] UDP:" -/var/log/iptables.log
+:msg, startswith, "UDP outlog:" -/var/log/iptables.log
```

Comment:
If we switch from the Docker json log driver to the journald log driver (but keep rsyslog), this setup would allow containers to spam our log file by starting their message with `UDP outlog:`. That's why last time I suggested that you change to an expression that would also filter on whether the message came via the kernel log facility, which would make it impossible for a container to spam the log file. If you get rid of this `:msg` line and replace it with this, it should do the trick:
`if $syslogfacility-text == 'kern' and $msg startswith 'UDP outlog: ' then -/var/log/iptables.log`